src/include/snapper/dialogs.rb

```diff
@@ -342,22 +335,36 @@ def CreateSnapshotPopup(pre_snapshots)
       created
     end
 
+
     # Popup for deleting existing snapshot
     # @return true if snapshot was deleted
     def DeleteSnapshotPopup(snapshot)
       snapshot = deep_copy(snapshot)
-      # yes/no popup question
-      if Popup.YesNo(
-          Builtins.sformat(
-            _("Really delete snapshot '%1'?"),
-            Ops.get_integer(snapshot, "num", 0)
-          )
-        )
-        return Snapper.DeleteSnapshot(snapshot)
+      num = Ops.get_integer(snapshot, "num", 0)
```

Comment:
Maybe, but the line was just moved I suppose and the task is not to replace all leftovers from the ycp to ruby conversion.